src/octod/api/common.d

```diff
@@ -51,7 +51,7 @@ void validateRepoString ( string repo )
     import std.regex;
     import std.exception : enforce;
 
-    static rgxRepo = regex(r"^[^/]+/[^/]+$");
+    static rgxRepo = regex(r"^[^@/]+/[^/]+$");
```

Comment:
Is this really related to this commit? Does it worth a comment explaining which is the expected form?

Reply:
I thought it was required for it to work but I realize now that I might have been wrong